packages/wallets/src/hooks/useIsRtl.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback, useEffect, useState } from 'react';
+
+const useIsRtl = () => {
+    const i18nLanguage = localStorage.getItem('i18n_language');
+
+    const checkRtl = useCallback(() => {
+        return i18nLanguage === 'AR';
+    }, [i18nLanguage]);
+
+    const [isRtl, setIsRtl] = useState<boolean>(() => checkRtl());
+
+    useEffect(() => {
+        setIsRtl(checkRtl());
+    }, [checkRtl]);
+
+    return isRtl;
+};
```

Comment:
Don't know, IMHO seems like here to much redundant logic as for me (extra useCallback, useState, useEffect), do we really need them?
In order to check that language is RTL we do the comparison: i18nLanguage === 'AR'.
So maybe we can refactor it to:

```suggestion
const useIsRtl = () => {
    const i18nLanguage = localStorage.getItem('i18n_language');

    return i18nLanguage === 'AR';
};
```

Reply:
yep this looks much better thanks 🙏  updated